exercises/src/components/exercise/preview.jsx

```diff
@@ -53,4 +55,4 @@ export default class ExerciseEditingPreview extends React.Component {
     );
   }
 
-}
+};
```

Comment:
is this right? classes dont take semicolons do they?

Reply:
they can.. I played with it a bit and it seems like ESLint gets a little confused about the decorator and export being attached to the class. If either of those are removed it stops suggesting a semi colon.